robocopkaka/hospital
# #161181014 Set up mailer in production

#### What does this PR do?
* Allows for the sending of emails in production
#### Description of Task to be completed?
* Add configuration to allow sending of emails through SendGrid
#### How should this be manually tested?
* Log in as a patient and create an appointment
* The admins should get a mail informing them about the appointment
#### What are the relevant pivotal tracker stories?
[#161181014](https://www.pivotaltracker.com/story/show/161181014)
#### Any background context you want to add?
N/A
#### Screenshots
![screen shot 2018-11-09 at 12 29 12 pm](https://user-images.githubusercontent.com/11176000/48260265-1ad1b100-e41b-11e8-8341-5535f7e1659d.png)
